mappings/doab.py

```diff
@@ -14,7 +15,27 @@ class DOABMapping(BaseMapping):
     def __init__(self, doab_record, namespaces):
         self.record = self.map_doab_record(doab_record, namespaces)
     
-    def map_doab_record(self, doab_record, namespaces) -> Record:
+    def map_doab_record(self, record, namespaces) -> Record:
+        header = record.find('.//header', namespaces=DSpaceService.ROOT_NAMESPACE)
+        deletion_flag = header is not None and header.get('status') == 'deleted'
+
+        if header is not None:
+            identifier = header.find('.//identifier', namespaces=DSpaceService.ROOT_NAMESPACE)
+            if identifier is not None:
+                source_id = identifier.text.split(':')[-1]
+
+        if deletion_flag and source_id:
+            delete_record=Record(
```

Comment:
Let's add a space around the equals sign. 

I think we can maybe just do `Record(source_id=source_id)` since the other values don't necessarily matter!